src/ui/common/src/components/execution/chip.tsx

```diff
@@ -0,0 +1,37 @@
+import Chip from '@mui/material/Chip';
+import React from 'react';
+
+import ExecutionStatus from '../../utils/shared';
```

Comment:
Is this a new component? I'm actually working on a similar component right now in my work for the Data and Workflow table views.

I don't want to hold your code up, but this is something we should consolidate in the future.